pkgs/tools/misc/zoxide/default.nix

```diff
@@ -10,13 +10,13 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "zoxide";
-  version = "0.9.2";
+  version = "unstable-20-11-2023";
```

Comment:
```suggestion
  version = "unstable-2023-11-20";
```

Reply:
I've also added a small explanation in the commit message as well.
let me know if I've missed anything.